timed/reports/management/commands/notify_reviewers_unverified.py

```diff
@@ -95,13 +109,14 @@ def _notify_reviewers(self, start, end, reports):
                         # we need start and end date in system format
                         'start': str(start),
                         'end': str(end),
+                        'message': message,
                         'reviewer': reviewer,
                         'protocol': settings.HOST_PROTOCOL,
                         'domain': settings.HOST_DOMAIN,
                     }, using='text'
                 )
 
-                mails.append((subject, body, from_email, [reviewer.email]))
+                mails.append((subject, body, from_email, [reviewer.email, cc]))
```

Comment:
this will be `To` in the mail but I guess this should be cc?

Reply:
I changed it as discussed.